ci/travis/format.sh

```diff
@@ -9,6 +9,7 @@ FLAKE8_VERSION_REQUIRED="3.7.7"
 YAPF_VERSION_REQUIRED="0.23.0"
 SHELLCHECK_VERSION_REQUIRED="0.7.1"
 MYPY_VERSION_REQUIRED="0.782"
+PYTHON_VERSION_REQUIRED="3.7.9"
```

Comment:
hmm I have 3.7.4 and it seems to work for me

Reply:
@richardliaw I could do something more specific with the versions (i.e. look at the minor version and do `>=` to "4" or something), but I genuinely don't know at what point lint would stop working. 

I feel like trying to be overly cautious with 'this may not work because it doesn't exactly equal' is better than having local lint fail.

Reply:
Hmm, ok - what's the output as a result of this change (on your local machine?)

Reply:
For version mismatch
```
WARNING: Ray uses python 3.7.9, You currently are using 3.7.0. This might generate different results.
```

Reply:
When I was using Python version `3.7.0`, `format.sh` failed to find this [patch](https://github.com/ray-project/ray/pull/14380/commits/f90541f2d13641aa436057cb8c8de29b4bb67d2a).

Reply:
the ci uses py3.6 though?